.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,8 +3,8 @@ quickly and easily:
 
  - [ ] Make sure the PR title is formatted like:
    `[BEAM-<Jira issue #>] Description of pull request`
- - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
-       Travis-CI on your fork and ensure the whole test matrix passes).
+ - [ ] Make sure tests pass via `mvn clean install -Pall-checks`. (Even better,
```

Comment:
I think there's value in having PR authors do the all-checks stuff, since things passing or failing on your own machine is easier to debug than in Jenkins. I think the value Jenkins is adding is the ITs, plus a safety net in case people haven't done their due diligence. I don't feel super strong about this though, so I'm open to pushback.